src/wasm-lib/grackle/src/native_functions.rs

```diff
@@ -32,6 +29,65 @@ impl<'a> Context<'a> {
     }
 }
 
+/// Unary operator macro to quickly create new bindings.
+macro_rules! define_unary {
+  () => {};
+  ($h:ident$( $r:ident)*) => {
```

Comment:
`h` is a confusing name for a variable -- isn't it just the function name, shouldn't it be called `fn_name` instead? and maybe `rest` instead of `r`? Same for `define_binary!` below.

Reply:
Will be changed in the follow up PR based on this one